ui/src/components/plugins/Plugin.vue

```diff
@@ -112,7 +112,10 @@
         watch: {
             $route: {
                 handler(newValue, _oldValue) {
-                    if (newValue.name.startsWith("plugins/")) {
+                    if (newValue.name === "plugins/list") {
+                        this.pluginType = undefined;
+                        this.version = undefined;
+                    } else if (newValue.name.startsWith("plugins/")) {
```

Comment:
Let's keep the scroll-up when we go back to the plugins home

```suggestion
                    } 
                    if (newValue.name.startsWith("plugins/")) {
```